modules/sandboxed-containers-collecting-data.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/troubleshooting-sandboxed-containers.adoc
+
+//This file contains UI elements and/or package names that need to be updated.
+
+[id="sandboxed-containers-collecting-data_{context}"]
+= About collecting {sandboxed-containers-first} data
+
+You can use the `oc adm must-gather` CLI command to collect information about your
```

Comment:
This is from the IBM style guide: 

> Translation consideration: To avoid possible translation problems, use complete sentences to introduce lists and not phrases. Use a colon at the end of the lead-in sentence unless another sentence intervenes between the lead-in sentence and the first list item. If a sentence intervenes, use a period to end both the lead-in sentence and the intervening sentence.

I suggest the following for this sentence: 

```suggestion
You can use the `oc adm must-gather` CLI command to collect information about your cluster. The following features and objects are associated with {sandboxed-containers-first}: 
```